gangplank/internal/spec/jobspec_test.go

```diff
@@ -17,26 +33,34 @@ func TestURL(t *testing.T) {
 	defer os.RemoveAll(tmpd) //nolint
 
 	cases := []struct {
-		repo    Repo
-		desc    string
-		wantErr bool
+		repo       Repo
+		desc       string
+		data       []byte
+		statusCode int
+		wantErr    bool
 	}{
 		{
-			desc:    "good repo",
-			repo:    Repo{URL: strPtr("http://mirrors.kernel.org/fedora-buffet/archive/fedora/linux/releases/30/Everything/source/tree/media.repo")},
-			wantErr: false,
+			desc:       "good repo",
+			data:       []byte("good repo"),
+			repo:       Repo{URL: strPtr("http://mirrors.kernel.org/fedora-buffet/archive/fedora/linux/releases/30/Everything/source/tree/media.repo")},
+			statusCode: 200,
+			wantErr:    false,
 		},
 		{
-			desc: "named repo",
+			desc:       "named repo",
+			data:       []byte("named repo"),
+			statusCode: 200,
 			repo: Repo{
 				Name: "test",
 				URL:  strPtr("http://mirrors.kernel.org/fedora-buffet/archive/fedora/linux/releases/30/Everything/source/tree/media.repo")},
 			wantErr: false,
 		},
 		{
-			desc:    "bad repo",
-			repo:    Repo{URL: strPtr("http://fedora.com/this/will/not/exist/no/really/it/shouldnt")},
-			wantErr: true,
+			desc:       "bad repo",
+			data:       nil,
+			statusCode: 404,
+			repo:       Repo{URL: strPtr("http://0.0.0.0/this/will/not/exist/no/really/it/shouldnt")},
```

Comment:
Shouldn't this either match the new one in rhcos-4.8 or all be set to `0.0.0.0`

Reply:
Since this is the main branch, we can backport it.

Reply:
Ok and what about changing `mirrors.kernel.org` to `0.0.0.0`, since there is no reason to have an actual working URL in the test case anymore?